modules/ui/src/endpoints.rs

```diff
@@ -5,3 +5,33 @@ use trustify_ui::{trustify_ui, UI};
 pub fn configure(config: &mut web::ServiceConfig, ui: &UI) {
     config.service(ResourceFiles::new("/", trustify_ui(ui)).resolve_not_found_to(""));
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use actix_web::{test, App};
+
+    #[test]
+    async fn test_ui_get() {
+        let test_ui = UI {
+            version: "".to_string(),
+            auth_required: "".to_string(),
+            oidc_server_url: "".to_string(),
+            oidc_client_id: "".to_string(),
+            oidc_scope: "".to_string(),
+            analytics_enabled: "".to_string(),
+            analytics_write_key: "".to_string(),
+        };
+
+        let app = test::init_service(App::new().configure(|cfg| configure(cfg, &test_ui))).await;
```

Comment:
Hm, I suspect this may still miss if someone adds endpoints differently. Sorta needs to be an "integration test" I suspect, hitting actual HTTP from the actual running server.

Reply:
the issue says:

>  It is not important that the UI is rendered or works. It should just check that it is served.

> hitting actual HTTP from the actual running server.

I don't disagree but that is other issue then ? :+1:

Reply:
>  I suspect this may still miss if someone adds endpoints differently. 

`let req = test::TestRequest::get().uri("/oooo").to_request();`

yeah it tests almost nothing 

`PASS [   0.010s] trustify-module-ui endpoints::test::test_ui_get`

Reply:
better to close this :+1: